### DISCUSSION TITLE
Capstone - Infobox different values

### DISCUSSION THREAD

**Discussion by MatthewBradleyLo** [teamalgoritma/community#715] [Q&A] [answered]
Saat saya masukan fungsi yang sama kepada infobox, nilainya beda dan salah 

![Screenshot (9)](https://github.com/teamalgoritma/community/assets/133225102/73a763ff-bd34-41b4-b5a7-df4c3082f8e2)
![Screenshot (8)](https://github.com/teamalgoritma/community/assets/133225102/5f18c560-cca7-40f8-89e9-a679ddf48549)
![Screenshot (7)](https://github.com/teamalgoritma/community/assets/133225102/252e845b-5011-4902-8cfc-3bd4472e1f1a)

**Comment by ranydwi** [comment, marked as answer]
Dear @MatthewBradleyLo 

Hal tersebut dikarenakan code yang Bapak masukan berbeda antara rmd dan juga ui.R Bapak. Coba bapak hapus `length()` pada setiap `value=` yang ada pada masing-masing infoBox Bapak. Hal ini karena saya rasa code tersebut tidak diperlukan. Cukup bagian `n_distinct(___)` saja ya Pak.

Semoga membantu.
Regards
